src/rust/ide/src/ide/initializer.rs

```diff
@@ -25,8 +25,8 @@ use ensogl::system::web::platform::Platform;
 //     download required version of Engine. This should be handled properly when implementing
 //     https://github.com/enso-org/ide/issues/1034
 const PROJECT_MANAGER_TIMEOUT_SEC     : u64  = 2 * 60 * 60;
-const ENGINE_VERSION_SUPPORTED        : &str = "^0.2.6";
-const ENGINE_VERSION_FOR_NEW_PROJECTS : &str = "0.2.6";
+const ENGINE_VERSION_SUPPORTED        : &str = "^0.2.8";
+const ENGINE_VERSION_FOR_NEW_PROJECTS : &str = "0.2.8";
```

Comment:
can we merge this config with the `src/js/lib/project-manager/src/build.ts` one? Changing this in 2 places seems error prone. At least, could you add comments in both places that are pointing to the other, please?

Reply:
I will add comment and create and issue for merging those constants.